questions.md

```diff
@@ -14,7 +21,8 @@ Because in 1.19.50, the [new execute syntax](https://learn.microsoft.com/en-us/m
 [Detect rightclick or leftclick (on an item)?](/wiki/questions/itemclick)  
 [Give a special item (Bedrock)?](/wiki/questions/giveitembedrock)  
 [Change an item while it's in the players inventory?](/wiki/questions/modifyinventory)  
-[Do custom crafting](/wiki/questions/customcrafting)
+[Do custom crafting?](/wiki/questions/customcrafting)  
+[Make a shop? / Buy items?](wiki/questions/shop)
```

Comment:
I'm not sure, but I think the link should start with `/`.

Reply:
you're correct, it should.